ui/apps/platform/src/Containers/VulnMgmt/List/Images/WatchedImagesDialog.tsx

```diff
@@ -131,7 +131,7 @@ const WatchedImagesDialog = ({ closeDialog }: WatchedImagesDialogProps): ReactEl
                     validationSchema={Yup.object({
                         imageTag: Yup.string()
                             .matches(
-                                /([a-z.]+\/)([a-z0-9-]+\/)([a-z0-9-@./]+)(?::[0-9a-z\-.]+)?/,
+                                /([a-z.]+\/)([a-z0-9-]+\/)([a-z0-9-@./]+)?(?::[0-9a-z\-.]+)?/,
```

Comment:
@boo question mark on the second group instead of the third group?

Reply:
^ @clickboo first comment has wrong mention, my bad

Reply:
Nope, the first group is the _beginning_ of the registry name. Check screenshots above. The third group is the repository.

Reply:
The code change does not seem to match the tester pattern in the pictures.

Reply:
Interestingly, ? on the second group also works. I guess I didn't peel my eyes enough.